Common/src/Injection/ProviderBase.cs

```diff
@@ -27,10 +27,10 @@
 
 namespace ArmoniK.Core.Common.Injection;
 
-public abstract class ProviderBase<T> : IHealthCheckProvider
+public abstract class ProviderBase<T> : IProviderBase<T>, IHealthCheckProvider
 {
   private readonly Func<Task<T>> builder_;
-  private          T?            object_;
+  private T? object_;
```

Comment:
I am pretty this does not follow the formatting rules.

Reply:
This is already what the indent produces